internal/daemon/controller/handlers/authmethods/authmethod_service_test.go

```diff
@@ -1550,3 +1599,491 @@ func TestCreate(t *testing.T) {
 		})
 	}
 }
+
+func TestListPagination(t *testing.T) {
+	// Set database read timeout to avoid duplicates in response
+	oldReadTimeout := globals.RefreshReadLookbackDuration
+	globals.RefreshReadLookbackDuration = 0
+	t.Cleanup(func() {
+		globals.RefreshReadLookbackDuration = oldReadTimeout
+	})
+	ctx := context.TODO()
```

Comment:
context.TODO() implies that this will eventually change. I think we always want Background here.
```suggestion
	ctx := context.Background()
```